crates/steel-core/src/tests/success/math.scm

```diff
@@ -0,0 +1,41 @@
+(define-syntax assert-equal!
+    (syntax-rules ()
+      ((_ expected actual)
+       (assert! (equal? expected actual)))))
+
+;; Number types
+(assert! (not (equal? 10 10.0)))
```

Comment:
You'll just need to add this file name here: https://github.com/mattwparas/steel/blob/master/crates/steel-core/src/tests/mod.rs#L89 - the test files inside the crate don't get automatically picked up unlike in some other spots (they probably should though. but that is a different issue)

Reply:
Done.